### PR TITLE
Update github-workflow.md

### PR DESCRIPTION
**Pull request by RPing**
Fixes #
give an alternative command combining `git fetch` and `git rebase` :p

### CODE REVIEW

**Comment by RPing** [comment]
ping @castrojo @guineveresaenger

**Comment by guineveresaenger** [comment]
Hi! Thanks for the ping and thank you for your patience! :) Also, thank you for your pull request and welcome to the community.

So...
I personally love me a good `git pull --rebase` but honestly, for a beginner-friendly doc, I like to keep it simple and give One Option That Works, and in this case the verbose version is both safer (no accidental `git pull`s without arguments) and makes it more obvious what's going on.

🤷 I'm happy to be convinced that including this is worthwhile so I'll leave some suggestions.

File: contributors/guide/github-workflow.md [hunk 1]
```diff
@@ -87,9 +92,12 @@ This workflow is process-specific; for quick start build instructions for [kuber
 # While on your myfeature branch
 git fetch upstream
 git rebase upstream/master
+
+# Or
+git pull --rebase upstream master
 ```
 
-Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull`
+Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull` without any options (like `--rebase`)
```
Comment (guineveresaenger) [inline contributors/guide/github-workflow.md:100]: I think if we do want to mention `git pull --rebase`, here should be the only place, i.e. "or another non-merge options such as `git pull --rebase`".

**Comment by RPing** [comment]
Yeah, I agree with you that there should be provided detailed commands in beginner's guide.
But after being more involved in these projects, there should be provided a more convenient way to simplify the flow.

**Comment by guineveresaenger** [comment]
@RPing sorry for the long delay! Could you re-word per my suggestion so it's clear it's an alternative for the more experienced git user?

**Comment by RPing** [comment]
@guineveresaenger I'm done.
And my editor automatically removes all spaces, is that okay?

File: contributors/guide/github-workflow.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull`
 does a merge, which leaves merge commits. These make the commit history messy
 and violate the principle that commits ought to be individually understandable
 and useful (see below). You can also consider changing your `.git/config` file via
-`git config branch.autoSetupRebase always` to change the behavior of `git pull`.
+`git config branch.autoSetupRebase always` to change the behavior of `git pull`, or another non-merge options such as `git pull --rebase`.
```
Comment (jdumars) [inline contributors/guide/github-workflow.md:96]: nit: "or another non-merge option such as ..."

**Comment by castrojo** [comment]
/approve

Not going to block on the nit, you can follow up later, thanks!

File: contributors/guide/github-workflow.md (hunk 2 above)
Comment (RPing) [reply contributors/guide/github-workflow.md:96]: Oops... I'll fix it. Thanks :+1:

**Comment by jdumars** [comment]
thanks for being so responsive @RPing! Let's get this merged.
/lgtm
/unhold

**Comment by guineveresaenger** [comment]
thank you @jdumars and @RPing and again sorry to be so slow on this.

**Pull request merged**